client/state/memberships/settings/actions.js

```diff
@@ -35,10 +31,7 @@ export const requestDisconnectStripeAccount = (
 		return wpcom.req
 			.get( `/me/connected_account/stripe/${ connectedAccountId }/disconnect` )
 			.then( () => {
-				dispatch( {
-					siteId,
-					type: MEMBERSHIPS_CONNECTED_ACCOUNTS_STRIPE_DISCONNECT_SUCCESS,
-				} );
+				dispatch( requestSettings( siteId ) );
 				dispatch( {
 					type: NOTICE_CREATE,
```

Comment:
btw this should use the `successNotice` action creator from _client/state/notices/actions_. There are many places in MailChimp/Memberships code that create the action objects directly.

Reply:
I agree, I thought I'd ignore it as it was unrelated, I will follow up on this in another PR. 👍

Reply:
Addressed in #47669